plugins/processors/starlark/testdata/pivot.star

```diff
@@ -0,0 +1,14 @@
+'''
+Pivots a key's value to be the key for another key.
+In this example it pivots the value of key `sensor`
+to be the key of the value in key `value`
+
+Input: measurement,tag_set sensor=001A0,value=111.48
+Output: measurement,tag_set 001A0=111.48
```

Comment:
love the input output section. We should make this standard for all examples.

Reply:
Got rid of the tags altogether as they don't play a role here